services/data_plane_manager/src/main/resources/application.properties

```diff
@@ -6,4 +6,11 @@ dataplane.isovs=true
 #####Logging configuration#####
 #logging.file.path=./
 #logging.file.name=data-plane-manager.log
-#logging.level.root=INFO
\ No newline at end of file
+#logging.level.root=INFO
+
+mq.type=pulsar
+#####Pulsar configuration#####
+pulsar.url=pulsar://192.168.131.131:6650
+pulsar.unicast.topic=unicast-topic1
+host.ip.to.group.topic.map=group-topic1:10.10.10.10,10.10.10.11 group-topic2:11.11.11.11,11.11.11.12
```

Comment:
This has the potential to become one of the largest configurations :-) 

Put it here is fine as of now but we might need to think of a different way to store it.